include/LabSound/core/Mixing.h

```diff
@@ -28,6 +28,13 @@ namespace lab
         BackLeft = 6,
         BackRight = 7
     };
+
+    static const int CHANNELS_MONO = 1;
```

Comment:
Suggest using `uint32_t` since channels will always be >= 0. Also thought about using an enum class here but these constants are used frequently and they'd all need to be casted.